remote-access/ssh/passwordless.md

```diff
@@ -55,21 +55,22 @@ ssh-rsa <REALLY LONG STRING OF RANDOM CHARACTERS> user@host
 <a name="copy-your-public-key-to-your-raspberry-pi"></a>
 ## Copy your public key to your Raspberry Pi
 
-To copy your public key to your Raspberry Pi, use the following command, on the computer you will be connecting from, to append the public key to your `authorized_keys` file on the Pi, sending it over SSH:
+
+On the computer which you will be connecting from, append the public key to your `authorized_keys` file on the Raspberry Pi by sending it over SSH:
```

Comment:
"On the computer which you will be connecting from, append the public key ..." - might sound like you're appending the public key _on_ the computer your connecting from? Does this alternative work better?
"On the computer which you will be connecting from, run the following command to append the public key ..."

And perhaps it's worth changing "your `authorized_keys` file" to "the `authorized_keys` file" ? :shrug:  (or "your user's `authorized_keys` file" ?)

Reply:
I think that needs the first word of the sentence change from 'on' to 'using'.